src/main/java/racingcargame/utill/RandomGenerator.java

```diff
@@ -0,0 +1,13 @@
+package racingcargame.utill;
+
+import java.util.Random;
+
+public class RandomGenerator implements RandomUtill{
```

Comment:
코드 포맷팅이 안되어있네요~
확인 부탁드려요!
```suggestion
public class RandomGenerator implements RandomUtill {
```

Reply:
감사합니다! 더 신경쓰겠습니다.